app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -87,7 +87,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:orientation="horizontal"
-        app:layout_constraintGuide_begin="300dp" />
+        app:layout_constraintGuide_begin="250dp" />
```

Comment:
dont use magical numbers. add the value to dimen.xml file.

Reply:
Yes, it was mine. Sure I'll create a ticket and would try to fix it